website/static/js/pages/base-page.js

```diff
@@ -88,6 +88,10 @@ $(document).on('click', '.panel-heading', function(){
     }
 });
 
+$(document).on('click', 'a[href="#"]', function(e) {
```

Comment:
There is some possibility, let me check btn-link. 



—
Sent from Mailbox

On Mon, Jul 13, 2015 at 10:26 AM, Steven Loria <notifications@github.com>
wrote:

>> @@ -88,6 +88,10 @@ $(document).on('click', '.panel-heading', function(){
>>      }
>>  });
>>  
>> +$(document).on('click', 'a[href="#"]', function(e) {
> Is there any possibility that this will intercept any of our other click handlers?
> Perhaps we shouldn't use `a` tags with `href="#"` and instead use `buttons` with the `btn-link` class.
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/CenterForOpenScience/osf.io/pull/3553/files#r34467769